### PR TITLE
update sanity query to be consistent with rails

### PR DESCRIPTION
**Pull request by zacjones93**
The sanity data coming in was different than the rails attribute so when the course data wasn't set in sanity, no instructor name would be shown. I converted the sanity data to be consistent with the rails data so now the objects merge instead of having separate `slug` and `url` attributes. We were only checking for `url` on the front end.

While making a sanity object for a course is a lot easier now, this shouldn't ever be an issue or something we should have to check

tweak

### CODE REVIEW

File: src/components/layouts/collection-page-layout.tsx [hunk 1]
```diff
@@ -257,8 +256,13 @@ const CollectionPageLayout: React.FunctionComponent<CoursePageLayoutProps> = ({
     [],
   ).map((lesson: LessonResource) => lesson.slug)
 
-  const {name, avatar_url, url: instructor_slug, bio_short, twitter} =
-    instructor || {}
+  const {
+    full_name: name,
+    avatar_url,
+    slug,
```
Comment (zacjones93) [inline src/components/layouts/collection-page-layout.tsx:262]: converted `name` to `full_name` in the sanity query to match the rails `full_name` attribute that comes in.

Comment (zacjones93) [reply src/components/layouts/collection-page-layout.tsx:262]: similarly, using `slug` to be consistent with rails

File: src/components/layouts/collection-page-layout.tsx [hunk 2]
```diff
@@ -420,7 +424,7 @@ const CollectionPageLayout: React.FunctionComponent<CoursePageLayoutProps> = ({
                   <InstructorProfile
                     name={name}
                     avatar_url={avatar_url}
-                    url={instructor_slug}
+                    url={slug}
```
Comment (zacjones93) [inline src/components/layouts/collection-page-layout.tsx:427]: use slug

File: src/lib/courses.ts [hunk 3]
```diff
@@ -34,9 +35,9 @@ const courseQuery = groq`
     'image': person->image.url
   },
 	'instructor': collaborators[]->[role == 'instructor'][0]{
-    'name': person->name,
+    'full_name': person->name,
     'avatar_url': person->image.url,
-    'url': person->slug.current
+    'slug': person->slug.current
```
Comment (zacjones93) [inline src/lib/courses.ts:40]: update sanity query for consistency with rails

**Review by joelhooks — APPROVED** [review]

**Pull request merged**